technische-universitat-munchen-controlling.csl

```diff
@@ -7,7 +7,7 @@
     <id>http://www.zotero.org/styles/technische-universitat-munchen-controlling</id>
     <link href="http://www.zotero.org/styles/technische-universitat-munchen-controlling" rel="self"/>
     <link href="http://www.zotero.org/styles/universitat-freiburg-geschichte" rel="template"/>
-    <link href="http://www.zotero.org/styles/hwr-berlin" rel="template"/>
+    <link href="http://www.zotero.org/styles/hochschule-fur-wirtschaft-und-recht-berlin" rel="template"/>
```

Comment:
Right, that's it. Just discovered that independently :P.

(the code for checking the template link is at https://github.com/citation-style-language/styles/blob/a6646ba5657dd22f3f4a534f6190b4a45be2895b/spec/styles_spec.rb#L26)